src/pages/drop/ticket/[id].tsx

```diff
@@ -0,0 +1,131 @@
+import { Badge, Box, Button, Text } from '@chakra-ui/react';
+
+import { CopyIcon, DeleteIcon } from '@/common/components/Icons';
+
+import { DropManager } from '@/modules/DropManager/DropManager';
+
+interface TicketDropResponse {
+  name: string;
+  links: {
+    id: number;
+    slug: string;
+    email: string;
+    name: string;
+    hasClaimed: boolean;
+  }[];
+}
+
+const tableColumns = [
+  { title: 'Name', selector: (row) => row.name },
+  { title: 'Email', selector: (row) => row.email },
+  { title: 'Link', selector: (row) => row.link },
+  { title: 'Claim Status', selector: (row) => row.hasClaimed },
```

Comment:
```suggestion
  { title: 'Status', selector: (row) => row.hasClaimed },
```
Figma:
<img width="205" alt="image" src="https://user-images.githubusercontent.com/19740800/212830133-b1963d18-a69d-4fdb-9e23-e9060dd2ffb2.png">

Reply:
Thanks for correcting this, I have updated here [8355a46](https://github.com/keypom/keypom-airfoil/pull/58/commits/8355a46dafac960422cea95582d9be228ff28e30)